sys/mips/timer.c

```diff
@@ -52,7 +52,7 @@ static int set_next_tick(mips_timer_state_t *state) {
   /* calculate next value of compare register based on timer period */
   do {
     state->compare.val += state->period_cntr;
-    mips32_set_c0(C0_COMPARE, state->compare.lo);
+    mips32_setcompare(state->compare.lo);
```

Comment:
```suggestion
    mips32_set_compare(state->compare.lo);
```

Reply:
`mips32_set_compare` - where it is defined? What is wrong with `mips32_setcompare`?

Reply:
Oh right, I didn't check the definition... 
Just noticed absolutely incoherent var/fun name, but it matches the naming scheme of definitions around